### DISCUSSION TITLE
Welcome to openrndr Discussions!

### DISCUSSION THREAD

**Discussion by edwinRNDR** [openrndr/openrndr#210] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hamoid** [comment]
Hi :-) I'm [aBe](https://hamoid.com), an artist + creative coder from Finland or Spain, depends on the day. Involved in Creative Code Berlin since 2013. 

I use OPENRNDR to explore generative design. I spent quite some time creating monochrome designs for pen plotters.

Here some random sketches from 2020 / 2021:

![openrndr-sketches-abe](https://user-images.githubusercontent.com/108264/111506157-a37a5d80-8749-11eb-9af0-8891dc96029d.png)

I'm curious about what everyone else is using OPENRNDR for :-)

**Comment by edwinRNDR** [comment]
I am @edwinRNDR, creative coder based in The Hague. I develop for- and maintain OPENRNDR. I actively use OPENRNDR to create interactive media installations at [RNDR](https://rnder.studio), and in [my own doodling](https://twitter.com/voorbeeld)
